editor/components/block-list/block.js

```diff
@@ -465,6 +478,7 @@ export class BlockListBlock extends Component {
 				...blockType.getEditWrapperProps( block.attributes ),
 			};
 		}
+		const blockElementId = `block-${ this.props.uid }`;
```

Comment:
I'm finding some general inconsistency:

- We destructure many props, but not `uid`
- We reference `uid` by `this.props` here, and by `block.uid` in the `BlockDraggable` below